src/explore-education-statistics-admin/package.json

```diff
@@ -179,6 +179,7 @@
     "last 1 version"
   ],
   "dependencies": {
+    "lodash": "^4.17.11",
```

Comment:
Seems like we're missing the corresponding changes to package-lock.json

Reply:
This was the only affected file when I ran:

`lerna add lodash src/explore-education-statistics-admin`

Reply:
Is there something else needed to force package-lock.json to update?

Reply:
Ah nevermind, it looks like it's already brought in as a top-level dep for some reason. I'm a bit suspicious that package-lock.json might be wrong though, but whatever I guess

Reply:
Okeydoke!